examples/simple-viewport/index.tsx

```diff
@@ -1,11 +1,29 @@
-import 'babel-polyfill'
-import React from 'react'
+import 'react-app-polyfill/ie11'
```

Comment:
Shipped with the example boilerplate, thought I'm thinking we probably don't need it?

Reply:
Agreed, let's remove :)